### DISCUSSION TITLE
PENCIL Support!

### DISCUSSION THREAD

**Discussion by diemuellermilch85** [standardnotes/forum#2754] [Suggestions]
Please, make a pencil support! This will be the feature for so many people, to use your software!! 
With no pencil support, there will be only a "hand full" of customers in the future. 
And we need a good alternative to Microsoft OneNote!

**Comment by JaspalSuri** [comment]
Hi @diemuellermilch85, thanks for the feedback. Supporting input for the Apple Pencil someday would be nice to have, but it isn't a part of our current plans.
